config/config.go

```diff
@@ -72,8 +82,15 @@ type Schema struct {
 	Attributes   map[string]*Attribute
 }
 
+type ReqSpec struct {
+	Prefix ReqPrefix
```

Comment:
Unfortunately Prefix is also used in the Req struct to refer to the very first part of the requirements (REQ or ASM). I think this could get a bit confusing in the future so we should rename one of them (I don't mind which, probably renaming the one in Req struct would be easier). Whatever happens, we could use REQ-TRAQ-SWL-25 to clearly document what each part of the requirement ID is called within the code.